src/App.jsx

```diff
@@ -13,46 +15,90 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const SORT_BY_LENGTH = 'length';
+const SORT_BY_ALHPABET = 'alpha';
+const SET_REVERSE = 'reverse';
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const sortGoods = (arrayOfGoods, { sortValue, reverseValue }) => {
+  const copyOfGoods = [...arrayOfGoods];
+
+  const sortArray = copyOfGoods.sort((goodA, goodB) => {
```

Comment:
you have two variables with same values here: **copyOfGoods === sortArray**.
**sort()** method mutate the array, so you could create only one variable and then just sort it, like this:
```
  const sortArray = [...arrayOfGoods];
  
  sortArray.sort((goodA, goodB) => {
  ...
  ...
  ...
```
It will be literaly the same but without extra **copyOfGoods** variable